Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -193,6 +221,17 @@ void AudioPane::LoadSettings()
 
   // DPL2
   m_dolby_pro_logic->setChecked(SConfig::GetInstance().bDPL2Decoder);
+  m_dolby_quality_slider->setValue(SConfig::GetInstance().m_dpl2_quality);
+  m_dolby_quality_latency_label->setText(GetDPL2ApproximateLatencyLabel(
+      static_cast<AudioCommon::DPL2Quality>(SConfig::GetInstance().m_dpl2_quality)));
+  if (AudioCommon::SupportsDPL2Decoder(current))
+  {
+    m_dolby_quality_label->setEnabled(m_dolby_pro_logic->isChecked());
```

Comment:
Any thoughts on extracting these setEnabled calls to a reusable function (`EnableDolbyQualityWidgets` maybe)? This feels very verbose.